src/instance/mod.rs

```diff
@@ -18,27 +18,39 @@ use data::{Map,Monster};
 
 pub mod management;
 
+// Computation of the duration of one so-called 'tick'. A 'tick' is a
+// unite in the discrete time of lycan. By default, its value is set
+// to about 16ms.
```

Comment:
The latest reason is mainly why I think it's best. You mentioned several times about issues of having things happening in the same tick. If we have a tick length long enough, we might be able to realize sooner the problems and people during beta testing will also be able to report the bugs more easily.